js/ui/appDisplay.js

```diff
@@ -272,6 +297,9 @@ const BaseAppView = new Lang.Class({
             if (!icon)
                 continue;
 
+            if (addedIds && addedIds.indexOf(itemId) != -1)
```

Comment:
Since `this._findAddedIcons()` always returns an array, you should not need the first condition here.

Reply:
Good point